charts/piraeus/templates/external-metrics-service.yaml

```diff
@@ -0,0 +1,39 @@
+{{ if (.Values.externalController).url }}
+---
+{{- $url := urlParse .Values.externalController.url -}}
+{{- $host := regexSplit ":" (get $url "host") -1 -}}
+{{- $ipAddr := index $host 0 -}}
+{{- $port := index $host 1 }}
+apiVersion: v1
+kind: Endpoints
+metadata:
+  name: {{ include "piraeus-operator.fullname" . }}-external-controller-metrics-service
+  labels:
+  {{- include "piraeus-operator.labels" . | nindent 4 }}
+    app.kubernetes.io/component: linstor-external-controller
+subsets:
+  - addresses:
+    - ip: {{ $ipAddr }}
```

Comment:
This could also be a DNS name. We should use `EndpointSlice` instead, which has a `addressType` which we could set. I guess we would somehow automatically decide if the hostname is actually an IP address or not :thinking:

Reply:
Added support for FQDN